### PR TITLE
145246 eliminate price information uos updated event

### PR DESCRIPTION
**Pull request by MitaliSingh16**
Eliminated the Unit of sale updated event and Price Change event to prevent pricing information from passing from SAP to ERP facade.
/erpfacade/priceinformation
/erpfacade/bulkpriceinformation
- Removed below Tables
  -unitpricechangeevents
  -pricechangemaste

- Modified below table - 
  -encevents
- Removed below Webjob 
    -PriceChangeEvent
    -Monitoring' 
- Cleanup webjob is modified.
- Also removed related FT's and UT's

### CODE REVIEW

File: tests/UKHO.ERPFacade.API.FunctionalTests/appsettings.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "WebhookInvalidPayloadFileName": "ID2_WebhookPayload_Invalid.JSON",
     "GeneratedXMLFolder": "ERPFacadeGeneratedXmlFiles",
     "UoSPayloadFileName": "UoS1_Pricing.JSON",
-    "PriceChangePayloadFileName": "PriceChangeInput.JSON",
+    //"PriceChangePayloadFileName": "PriceChangeInput.JSON",
```
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/appsettings.json:8]: please remove commented code

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs [hunk 2]
```diff
@@ -912,7 +907,7 @@ public static int CalculateTotalNumberOfActions(JsonPayloadHelper jsonPayload)
 
         public static int CalculateNewCellCount(JsonPayloadHelper jsonPayload)
         {
-            var obj = jsonPayload;
+            JsonPayloadHelper obj = jsonPayload;
```
Comment (Sadha1501493) [inline tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs:910]: Please keep it var only.

**Review by Sadha1501493 — CHANGES_REQUESTED** [review]

File: tests/UKHO.ERPFacade.API.FunctionalTests/appsettings.json (hunk 1 above)
Comment (MitaliSingh16) [reply tests/UKHO.ERPFacade.API.FunctionalTests/appsettings.json]: Done

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs (hunk 2 above)
Comment (MitaliSingh16) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/SAPXmlHelper.cs]: Done

**Review by pravinvarade10801 — APPROVED** [review]
LGTM

**Review by Sadha1501493 — APPROVED** [review]
LGTM

**Review by Vishal14583 — CHANGES_REQUESTED** [review]
Added few review comments.

File: Deployment/templates/continuous-testing.yml [hunk 3]
```diff
@@ -21,15 +21,15 @@ steps:
...
-      azureSubscription: "ERP-Facade-Dev-A.011.05.12"
-      scriptType: 'pscore'
-      scriptLocation: 'scriptPath'
-      scriptPath: "$(Pipeline.Workspace)/terraformartifact/set_mock_configuration.ps1"
-      arguments: '-ftrunning "true" -mockresourcegroup $(mockWebAppResourceGroup) -mockwebappname $(mockWebAppName)'
+  # - task: AzureCLI@2
+  #   displayName: "Set FT Configuration Application Settings for Mock Webapp"
+  #   condition: and(succeeded(), eq(variables['Environment.Name'], 'ERP-QA'))
+  #   inputs:
+  #     azureSubscription: "ERP-Facade-Dev-A.011.05.12"
```
[the remark this answers is not in the document]
Comment (amogh14893) [reply Deployment/templates/continuous-testing.yml]: Removed commented code

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/RoSXMLHelper.cs [hunk 4]
```diff
@@ -11,12 +11,12 @@ namespace UKHO.ERPFacade.API.FunctionalTests.Helpers
     [TestFixture]
     public class RoSXmlHelper
     {
-        private static JsonInputRoSWebhookEvent jsonPayload;
+        private static JsonInputRoSWebhookEvent s_jsonPayload;
```
[the remark this answers is not in the document]
Comment (MitaliSingh16) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/RoSXMLHelper.cs]: done

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/JSONHelper.cs [hunk 5]
```diff
@@ -7,50 +7,15 @@ namespace UKHO.ERPFacade.API.FunctionalTests.Helpers
 {
     public class JsonHelper
     {
-        private static readonly string _projectDir = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory));
+        private static readonly string s_projectDir = Path.GetFullPath(Path.Combine(Environment.CurrentDirectory));
```
[the remark this answers is not in the document]
Comment (MitaliSingh16) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/JSONHelper.cs]: done

**Review by Vishal14583 — APPROVED** [review]
LGTM

**Review by Vishal14583 — APPROVED** [review]
LGTM

**Review by jrippington — APPROVED** [review]
Looks good

**Pull request merged**